### ISSUE TITLE
About semantic categories

### ISSUE THREAD

**Issue by Ian-dddd** [4DVLab/IDKB#1]
Thank you for the greak work!
Are you planning to release the semantic category for each Q?

**Comment by yuhanglu2000** [comment]
Certainly, we will soon optimize the data format and provide comprehensive usage guides. I try to complete this by the end of the month.

**Comment by Ian-dddd** [comment]
Thanks!

**Comment by yuhanglu2000** [comment]
Hello, we’ve optimized the structure of the data dictionary in the dataset. Please refer to the new link. The semantic categories are provided as key-value pairs.
